package.json

```diff
@@ -25,6 +25,7 @@
     "test:ci": "cross-env CI=true pnpm -r --stream --filter !test-fails --filter !test-browser run test --allowOnly",
     "test:ci:single-thread": "cross-env CI=true pnpm -r --stream --filter !test-fails run test --allowOnly --no-threads",
     "typecheck": "tsc --noEmit",
+    "typecheck:why": "tsc --noEmit --explainFiles > explainTypes.txt",
```

Comment:
👀? Cool magic good to know!

Reply:
Wanted to keep it in case someone will mess up the types again 👀